Tasks/UniversalPackagesV0/package.json

```diff
@@ -21,7 +21,7 @@
         "@types/node": "^16.11.39",
         "@types/uuid": "^8.3.0",
         "azure-pipelines-task-lib": "^4.13.0",
-        "azure-pipelines-tasks-packaging-common": "^3.242.0",
```

Comment:
Did you intentionally lock this to exactly this version instead of allowing it to float?

Reply:
replied in that comment